Gordon360/Models/ViewModels/RecIM/CreateActivityViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Gordon360.Models.CCT;
+using System;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class CreateActivityViewModel
```

Comment:
I think we determined in the other PR that this would be best suited as `ActivityUploadViewModel`, then it can be used for create and update operations.